libcxx/test/std/atomics/atomics.types.operations/atomics.types.operations.req/atomic_is_lock_free.pass.cpp

```diff
@@ -27,8 +27,12 @@ struct TestFn {
   void operator()() const {
     typedef std::atomic<T> A;
     T t = T();
+
```

Comment:
unrelated?

Reply:
It made the test a bit clearer since we are testing for non-volatile and volatile symmetrically.